meorg_client/client.py

```diff
@@ -503,3 +515,19 @@ def success(self) -> bool:
             True if successful, False otherwise.
         """
         return self.last_response.status_code in mcc.HTTP_STATUS_SUCCESS_RANGE
+
```

Comment:
Again, got lost between commits. Restored.

Reply:
Small change, but I think we can map `response` to make it a bit cleaner

```py
file_ids = map(lambda response: response.get("data").get("files")[0].get("file"), responses)
```

Or maybe you are using a similar utility function (not sure about this) like the following:

https://github.com/CABLE-LSM/meorg_client/blob/397a3004d76864bdc0c2a2202f5858bf58ed9adf/meorg_client/utilities.py#L89